src/io_array.ml

```diff
@@ -89,21 +89,13 @@ module Make (IO : Io.S) (Elt : ELT) :
               ( div b.low_off Elt.encoded_sizeL,
                 div b.high_off Elt.encoded_sizeL )
           in
-          if low >= low_buf && high <= high_buf then
-            Log.debug (fun m ->
```

Comment:
Personally, I found these debug messages helpful for tracking down issues in the `interpolation_search`. I assume you removed them because they were too noisy?